### PR TITLE
Added the menu

### PR DESCRIPTION
**Pull request by JOdiemo**
Use basic JavaScript syntax.
Use JavaScript to manipulate DOM elements.
Use JavaScript events.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+                <li><a href="#about" class="nav">About</a></li>
+                <li><a href="#contact" class="nav">Contact</a></li>
+            </ul>
+        </nav>
+        <div class="navigation"  onclick = "showMenu()">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+        </div>           
+    </header>
```
Comment (ShadyShawkat) [inline index.html]: - [ ] Kindly increase the width of the mobile menu to be the entire width of the screen, to match the Figma template.
- [ ] Please eliminate the horizontal scroll in the mobile menu to make your website more responsive.
- [ ] Please use the same font style (font family) as the Figma design.

| Your design | Figma design |
| --- | --- |
| image | image |

Comment (ShadyShawkat) [inline index.html]: - [ ] Kindly adjust the spacing and size of the mobile menu header on the mobile view to match the Figma design.

| Your design | Figma design |
| --- | --- |
| image | image |

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @JOdiemo, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,565 @@
...
+  margin-left: -0.1px;
+  top: -0.1px;
+  width: 100vw;
+  height: 100vh;
+  transition: all 0.3s ease-in-out;
+  background: rgba(96, 112, 255, 0.8);
+  backdrop-filter: blur(5px);
+  overflow: hidden;
+  right: 0;
+}
```
Comment (leonardodiasb) [inline styles.css]: - [OPTIONAL] If the user scrolls down when the menu is open, it is possible to see the page content and leave the menu view. You could change the position to "fixed" to change that behavior.
- [ ] As the previous Code Reviewer mentioned, please eliminate the horizontal scroll in the mobile menu to make your website more responsive. 
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @JOdiemo , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
...
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @JOdiemo  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Good work implementing changes requested by the previous reviewers 👍 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**